bakery/checker_test.go

```diff
@@ -851,8 +847,8 @@ func (c *client) doAny(ctxt context.Context, svc *service, ops ...bakery.Op) (*b
 }
 
 // capability returns a capability macaroon for the given operations.
-func (c *client) capability(ctxt context.Context, svc *service, ops ...bakery.Op) (*macaroon.Macaroon, error) {
-	var m *macaroon.Macaroon
+func (c *client) capability(ctxt context.Context, svc *service, ops ...bakery.Op) (*bakery.Macaroon, error) {
```

Comment:
I am, but I'm not changing all existing uses right now. I'll do a global replace after this PR at some point. There are hundreds of uses and this PR is already way too big.